forge-gui/res/cardsfolder/upcoming/norns_fetchling.txt

```diff
@@ -0,0 +1,13 @@
+Name:Norn's Fetchling
+ManaCost:1 W
+Types:Creature Phyrexian Beast
+PT:1/1
+K:Toxic:1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigBranch | TriggerDescription$ When CARDNAME enters the battlefield, conjure a card named Plains into your hand. If an opponent has three or more poison counters, you may seek a nonland card instead.  
```

Comment:
Does this need PrecostDesc$ Corrupted — ?

Reply:
I think this only works on activated abilities with a cost